package-files/systemd/infra.service

```
@@ -0,0 +1,18 @@
+[Unit]
+Description=Modern infrastructure access.
```

I guess this should be a description of the service rather than of Infra?

```suggestion
Description=Run the infra ssh connector
```

Or something like that?